public/locales/en/translation.json

```diff
@@ -0,0 +1,138 @@
+{
+  "commoncomponents": {
+    "button": {
+      "change": {
+        "language": "Change language"
+      },
+      "login": {
+        "dev": {
+          "account": {
+            "login": "Login with Dev account"
+          }
+        },
+        "msal": {
+          "login": "Login with Active Directory (MSAL)"
+        },
+        "regular": {
+          "login": "Sign In"
```

Comment:
Just a detail but if there are no real difference between "Sign in" and "Log in", why not use Sign in everywhere ? To stay consistent